zulip/integrations/twitter/twitter-bot

```diff
@@ -164,4 +165,10 @@ for status in statuses[::-1][:options.limit_tweets]:
     else:
         since_id = status.id
 
-write_config(config, since_id, user_id)
+if 'twitter' not in config_internal.sections():
+    config_internal.add_section('twitter')
+config_internal.set('twitter', 'since_id', str(since_id))
+config_internal.set('twitter', 'user_id', str(user))
+
+write_config(config_internal, CONFIGFILE_INTERNAL)
+write_config(config, CONFIGFILE)
```

Comment:
Ha, you're right, we don't. I was too fixed on just implementing the second file :P